### ISSUE TITLE
aria labels missing on tab headings for comment preview "edit" and "preview"

### ISSUE THREAD

**Issue by rdbartlett** [loomio/loomio#3883]
![image](https://cloud.githubusercontent.com/assets/970124/20414867/9b5f6e4e-ad99-11e6-889c-c3a10c5c7198.png)

Just some missing text. But the tabs work nicely for a11y users :)

**Comment by gdpelican** [comment]
This is now fixed and will go in the next deploy
